Source/WebCore/css/DOMMatrixReadOnly.cpp

```diff
@@ -227,21 +228,15 @@ ExceptionOr<DOMMatrixReadOnly::AbstractMatrix> DOMMatrixReadOnly::parseStringInt
     if (string.isEmpty())
         return AbstractMatrix { };
 
-    auto styleDeclaration = MutableStyleProperties::create();
-    if (CSSParser::parseValue(styleDeclaration, CSSPropertyTransform, string, IsImportant::Yes, HTMLStandardMode) == CSSParser::ParseResult::Error)
+    CSSToLengthConversionData conversionData { };
```

Comment:
Surprised the braces are needed here. Should we initialize more in the `struct` or `class` itself instead?

Reply:
I don't think they are needed. I'll try to remove them.